docs/docs-site/content/developer/components/parsers/_index.md

```diff
@@ -24,15 +24,9 @@ Refer the [SQL Parser Documentation](/developer/development/#sql-parsers) for ho
       <option value="calciteAutocompleteParser">Calcite Autocomplete Parser</option>
       <option value="calciteSyntaxParser">Calcite Syntax Error Parser</option>
       <option disabled> </option>
-      <option value="elasticsearchAutocompleteParser">Elasticsearch Autocomplete Parser</option>
-      <option value="elasticsearchSyntaxParser">Elasticsearch Syntax Error Parser</option>
-      <option disabled> </option>
       <option value="phoenixAutocompleteParser">Phoenix Autocomplete Parser</option>
       <option value="phoenixSyntaxParser">Phoenix Syntax Error Parser</option>
       <option disabled> </option>
-      <option value="druidAutocompleteParser">Druid Autocomplete Parser</option>
-      <option value="druidSyntaxParser">Druid Syntax Error Parser</option>
-      <option disabled> </option>
```

Comment:
With this change we are also adding a dask parser and autocomplete, should we not add that to the list here?

Reply:
Thanks for pointing it out, didn't know. Adding.